src/main/kotlin/build/buf/intellij/index/BufIndexes.kt

```diff
@@ -38,7 +38,7 @@ object BufIndexes {
     fun getProjectModuleKeys(project: Project): Collection<ModuleKey> = getProjectIndexKeys(MODULE_KEY_INDEX_ID, project)
 
     private fun <K : Any> getProjectIndexKeys(indexId: ID<K, Void>, project: Project): Collection<K> {
-        val scope = GlobalSearchScope.allScope(project)
+        val scope = GlobalSearchScope.projectScope(project)
```

Comment:
We don't want to index or return `buf.yaml` or `buf.lock` entries for our dependencies - all dependencies (including transitive ones) are found in `buf.lock` files.